emma-examples/emma-examples-flink/src/test/scala/org/emmalanguage/examples/ml/classification/FlinkNaiveBayesIntegrationSpec.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright © 2014 TU Berlin (emma@dima.tu-berlin.de)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.emmalanguage
+package examples.ml.classification
+
+import api.Meta.Projections._
+import api._
+import examples.ml.model._
+
+import breeze.linalg.Vector
+import org.apache.flink.api.scala.ExecutionEnvironment
+
+class FlinkNaiveBayesIntegrationSpec extends BaseNaiveBayesIntegrationSpec {
+
+  def naiveBayes(input: String, lambda: Double, modelType: MType): Set[Model] =
+    /*emma.onFlink*/ {
```

Comment:
You can enable this after removing the `Meta` context bound from case classes in `examples.ml.model` (those are not really needed). I guess Flink didn't expect a case class with multiple parameter lists (introduced by the implicit context bound) because case classes generate getters, setters and `hashCode`/`equals` only for the first parameter list.